CONTRIBUTE.md

```diff
@@ -0,0 +1,30 @@
+## How to contribute to Next.js for Acquia CMS
+
+- You can use the web server of your choice.
+
+1. Have Acquia CMS installed from https://github.com/acquia/next-acms/blob/main/README.md#installation-acquia-cms with your Node environment variables copied to your clipboard.
+
+![image](https://user-images.githubusercontent.com/35243020/178577853-d6e91755-e6e3-4d62-9e56-ef7b7566c2c1.png)
+
+3. Fork the `acquia/next-acms` project repo and pull it down to your local machine (outside of the ACMS directory is recommended) and add the original repo as the upstream.
+
+```
+$ cd my_name/next-acms
+$ git remote add upstream https://github.com/acquia/next-acms
+```
+
+4. Go into `starters/basic-starter/.env.local` in your forked repo and paste in your environment variables from step 1.
```

Comment:
Does this file exist already? If not, maybe we should just tell them to create the file with the env vars.